apps/web/components/pages/task/details-section/blocks/task-progress.tsx

```diff
@@ -82,56 +126,111 @@ const TaskProgress = () => {
 					isAuthUser={true}
 					activeAuthTask={true}
 					showPercents={true}
-					memberInfo={memberInfo}
+					// memberInfo={memberInfo}
 				/>
 			</TaskRow>
 			<TaskRow labelTitle={trans.TOTAL_TIME} wrapperClassName="mb-3">
 				<div className="not-italic font-semibold text-xs leading-[140%] tracking-[-0.02em] text-[#282048] dark:text-white">
-					2h : 12m
+					{userTotalTime.hours}h : {userTotalTime?.minutes}m
 				</div>
 			</TaskRow>
 			<TaskRow labelTitle={trans.TIME_TODAY} wrapperClassName="mb-3">
 				<div className="not-italic font-semibold text-xs leading-[140%] tracking-[-0.02em] text-[#282048] dark:text-white">
-					1h : 10m
+					{userTotalTimeToday.hours}h : {userTotalTimeToday.minutes}m
 				</div>
 			</TaskRow>
 			<TaskRow labelTitle={trans.TOTAL_GROUP_TIME} wrapperClassName="mb-3">
 				<Disclosure>
 					{({ open }) => (
 						<div className="flex flex-col w-full">
-							<Disclosure.Button className="flex justify-between items-center w-full">
+							{task?.members !== undefined && task?.members.length > 1 ? (
```

Comment:
Insted of `task?.members !== undefined && task?.members.length > 1` we can use `task?.members && task?.members.length`